api/INSTALL.md

```diff
@@ -0,0 +1,13 @@
+# SimpleBook PDF Printing API
+
+Simplebook uses our node.js command line app in the subdirectory `mw2pdf/`.
```

Comment:
I have some updates for the installation instructions (node and python both) which I'll insert in a separate PR after merging this.